pupil_src/shared_modules/marker_tracker_3d/storage/controller_storage.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import os
+
+import numpy as np
+
+from marker_tracker_3d import worker
+from observable import Observable
+
+logger = logging.getLogger(__name__)
+
+
+class ControllerStorage(Observable):
+    def __init__(self, min_marker_perimeter, save_path):
+        self.min_marker_perimeter = min_marker_perimeter  # adjustable in UI
+
+        self.save_path = save_path
+
+        self._set_to_default_values()
+
+    def _set_to_default_values(self):
+        # marker_id_to_detections is for drawing in 2d window
```

Comment:
I would go with just `# for drawing in 2d window`. It's clear what attribute you are talking about, so you don't need to name it again. The problem with such comments is, that people forget to update them when they change variable names and this is very confusing.